code/drasil-lang/Language/Drasil/Chunk/Eq.hs

```diff
@@ -1,24 +1,31 @@
 {-# LANGUAGE TemplateHaskell #-}
-module Language.Drasil.Chunk.Eq (QDefinition, fromEqn, fromEqn', fromEqnSt, 
+module Language.Drasil.Chunk.Eq (QDefinition, fromEqn, fromEqn', fromEqnSt,
   fromEqnSt', equat, ec) where
 
 import Control.Lens ((^.), makeLenses, view)
 import Language.Drasil.Chunk.UnitDefn (unitWrapper, MayHaveUnit(getUnit))
 
 import Language.Drasil.Classes.Core (HasUID(uid), HasSymbol(symbol))
 import Language.Drasil.Classes (NamedIdea(term), Idea(getA),
-  IsUnit, DefiningExpr(defnExpr), Definition(defn), Quantity, HasSpace(typ))
+  IsUnit, DefiningExpr(defnExpr), Definition(defn), Quantity, HasSpace(typ), ExprRelat(relat), ConceptDomain(cdom))
 import Language.Drasil.Chunk.Quantity (QuantityDict, mkQuant, mkQuant', qw)
 
-import Language.Drasil.Expr (Expr)
+import Language.Drasil.Expr (Expr, ($=))
+import Language.Drasil.Expr.Math (sy)
 import Language.Drasil.NounPhrase (NP)
 import Language.Drasil.Space (Space)
 import Language.Drasil.Sentence (Sentence(EmptyS))
 import Language.Drasil.Stages (Stage)
 import Language.Drasil.Symbol (Symbol)
+import Language.Drasil.UID (UID)
 
 -- | A QDefinition is a 'Quantity' with a defining expression, and a definition
-data QDefinition = EC { _qua :: QuantityDict , _defn' :: Sentence, _equat :: Expr }
+data QDefinition = EC
+  { _qua :: QuantityDict
+  , _defn' :: Sentence
+  , _equat :: Expr
+  , cd :: [UID] -- TODO: FIXME, there is no way to set this right now!!
```

Comment:
@JacquesCarette I imported this as it is from the Qdef-in-instmod branch, before marking this PR ready for review (which is a great feature that I didn't know we could do on GitHub!), I wanted to ask if there is a specific way I should be going about adding this as a field to the smart constructors (should it be added as a field but derived from anything automatically, or is it intended to be manually set)?

Reply:
`cd` is intended to be manually set. Once `ModelKind` is implemented, as a second step, we'll want to change the combinators to require the 'concept domain'. But first, we'll have to make sure that the concept domain(s) of all of our examples exist!

The best way to deal with that is probably to create multiple new issues to deal with that. First some issues based on changing each new combinator to accept domains, then issues around adding the missing domains, then issues around fixing the examples to use these new combinators, and finally issues to remove the old combinators. So this can easily spawn a dozen new PRs.